### ISSUE TITLE
Warning for deprecated command in config_flow.py

### ISSUE THREAD

**Issue by smarthomefamilyverrips** [rospogrigio/localtuya#1203]
Logger: homeassistant.helpers.frame
Source: helpers/frame.py:77
First occurred: 14:02:48 (1 occurrences)
Last logged: 14:02:48

Detected integration that uses deprecated `async_get_registry` to access entity registry, use async_get instead. Please report issue to the custom integration author for localtuya using this method at custom_components/localtuya/config_flow.py, line 682: ent_reg = await er.async_get_registry(self.hass)

**Comment by jocamane** [comment]
Any news about this

**Comment by smarthomefamilyverrips** [comment]
@jocamane it looks that is solved with latest localtuya update.

**Comment by jocamane** [comment]
> @jocamane it looks that is solved with latest localtuya update.

Hi, in my case I have the version 5.0.0. it's the last version ?

If it is the last version the problem not resolved.

**Comment by smarthomefamilyverrips** [comment]
@jocamane yes it is, at me the message not showing anymore. Too bad is not resolved then

**Comment by frederickcwong** [comment]
Hi - the message also show up today in my container log. I am running version 5.0.0 as well.
